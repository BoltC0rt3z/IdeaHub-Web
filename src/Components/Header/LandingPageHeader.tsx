import React from 'react';
import { Row } from 'antd';

import Header from './Header';

import styles from './LandingPageHeader.module.scss';

interface Props {
  hasAccount: boolean;
}

const LandingPageHeader: React.FC<Props> = (props) => {
  const { hasAccount } = props;

  return (
    <Row className={styles.header}>
      <Header title="IdeaHub" hasAccount={hasAccount} />
    </Row>
  );
};

export default LandingPageHeader;
