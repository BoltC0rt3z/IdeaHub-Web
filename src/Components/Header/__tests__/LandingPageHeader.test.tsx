import React from 'react';
import { ReactWrapper, mount } from 'enzyme';
import { BrowserRouter as Router } from 'react-router-dom';

import LandingPageHeader from '../LandingPageHeader';

describe('Tests for landing page', () => {
  const wrapper: ReactWrapper = mount(
    <Router>
      <LandingPageHeader hasAccount={false} />
    </Router>,
  );

  it('should render landing page header without crushing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
