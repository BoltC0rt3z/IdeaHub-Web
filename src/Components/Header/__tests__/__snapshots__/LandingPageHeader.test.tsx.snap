// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for landing page should render landing page header without crushing 1`] = `ReactWrapper {}`;
